src/mscorlib/src/System/Resources/ResourceManager.cs

```diff
@@ -99,9 +98,8 @@ internal class PRIExceptionInfo
     // RM will need to use file-based or manifest-based resources, and 
     // afterwards refers to this through the interface IResourceGroveler.
     // 
-    // Serialization Compat: Ideally, we could have refactored further but
-    // this would have broken serialization compat. For example, the
-    // ResourceManager member UseManifest and UseSatelliteAssem are no 
+    // TODO: more serialization cleanup as type isn't serializable anymore.
```

Comment:
I am not aware of any clean up needed in ReourceManager nor aware of any issue around the serialization. it looks the comment is really old comment.

---
In reply to: [131532032](https://github.com/dotnet/coreclr/pull/13242#discussion_r131532032) [](ancestors = 131532032)